src/modules/status.rs

```diff
@@ -52,6 +71,60 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     }
 }
 
+fn status_common_meaning(ex: u32) -> Option<String> {
```

Comment:
I'd prefer if the below functions all used `&str` instead of `String` and always returned `None` if no match is found instead of either `None` or and empty string.

Reply:
Done I introduced a new variable for the case where no meaning is found